### PR TITLE
example on how to add it github for a pr

### PR DESCRIPTION
**Pull request by smontana30**

### CODE REVIEW

File: functions/src/Models/Events.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export class Events {
     location:  string;
     points: string;
     committee: string;
-    type: any;
```
Comment (BautistaBertolami) [inline functions/src/Models/Events.ts:10]: I don't see ANY problems with this

**Comment by jeffjramos21** [comment]
Looks sick 👍

File: functions/src/Models/Events.ts (hunk 1 above)
Comment (smontana30) [reply functions/src/Models/Events.ts:10]: i dont like your comment

Comment (BautistaBertolami) [reply functions/src/Models/Events.ts:10]: Cool

File: functions/src/Models/Events.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ export class Events {
     location:  string;
     points: string;
     committee: string;
```
Comment (jeffjramos21) [inline functions/src/Models/Events.ts:9]: dope

**Pull request merged**